contributors/devel/sig-scalability/hollow-node-sts.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: v1
+kind: Service
```

Comment:
I'm curious why you're creating a service for the hollow-nodes? (assuming no traffic ideally should be sent to these hollow-nodes)